frontend/src/pages/Batch.tsx

```diff
@@ -90,11 +86,70 @@ const Batch: React.FC = () => {
     );
   }, [batch]);
 
+  function getSparkSubmitArg(key: string, value: string) {
+    if (value) {
+      return ' ' + key + ' ' + value;
+    } else {
+      return '';
+    }
+  }
+
+  const sparkSubmitStr = useMemo(() => {
+    if (!batch) {
+      return null;
+    }
+
+    return (
+      <Code className={styles.logs}>
+        spark-submit
+        {getSparkSubmitArg('--name', batch.submitParams.name)}
+        {getSparkSubmitArg('--driver-cores', batch.submitParams.driverCores.toString())}
+        {getSparkSubmitArg('--driver-memory', batch.submitParams.driverMemory)}
+        {getSparkSubmitArg('--num-executors', batch.submitParams.numExecutors.toString())}
+        {getSparkSubmitArg('--executor-cores', batch.submitParams.executorCores.toString())}
+        {getSparkSubmitArg('--executor-memory', batch.submitParams.executorMemory)}
+        {getSparkSubmitArg('--py-files', batch.submitParams.pyFiles.join(','))}
+        {getSparkSubmitArg('--archives', batch.submitParams.archives.join(','))}
+        {getSparkSubmitArg('--files', batch.submitParams.files.join(','))}
+        {getSparkSubmitArg('--jars', batch.submitParams.jars.join(','))}
+        {Object.entries(batch.submitParams.conf).map(([name, val]) => (
+          <span key={name}>
+            --conf {name} {val + ' '}
+          </span>
+        ))}
+        {' ' + batch.submitParams.file}
+        {' ' + batch.submitParams.args.join(' ')}
+      </Code>
+    );
+  }, [batch]);
+
+  const logsStr = useMemo(() => {
+    if (!logs) {
+      return null;
+    }
+
+    if (logs.log.startsWith('http')) {
+      return <a href={logs.log}>{logs?.log}</a>;
+    } else {
+      return <Code className={styles.logs}>{logs.log}</Code>;
+    }
+  }, [logs]);
+
   return (
     <div className={styles.batch}>
       <PageHeading>Batch {id}</PageHeading>
-      <Code className={styles.logs}>{logs?.log}</Code>
+      <Box mt="5">
+        <b>Logs:</b>
```

Comment:
`<Box textStyle='caption'>` or something similar should be used instead of `b` tag